collector/monitors/slashing_monitor.go

```diff
@@ -203,3 +213,10 @@ func (r *V1ValidatorsRepository) GetValidatorsAddresses(ctx context.Context) ([]
 
 	return hubResp.Validators, nil
 }
+
+
+func (r *V1ValidatorsRepository) DisplayName(pubkey string) string {
+	// TODO: implement the real method to match pubkey with display name
+	valName := []rune(pubkey)
```

Comment:
Why not use moniker from validator info endpoint? I think, it's clearer than just a few first characters of a validator address

Reply:
done
![Screenshot_20210805_174733](https://user-images.githubusercontent.com/62722506/128371158-f50d3fa4-2bff-40e4-8f74-9b53d9d174b8.png)